### ISSUE TITLE
TypeError: goog.dom.htmlToDocumentFragment is not a function

### ISSUE THREAD

**Issue by jiyinyiyong** [adzerk-oss/boot-reload#79]
I heard it's deleted: http://stackoverflow.com/questions/12318667/closure-library-dom-node-from-html-text#comment63223933_12323597
I'm running latest ClojureScript, `1.9.76`, there was a Google Closure Compiler update https://github.com/clojure/clojurescript/blob/master/changes.md#changes .
Tried `1.9.36` and it is fine.

**Comment by martinklepsch** [comment]
It's deprecated but probably still part of the library. I guess we could use the `SafeHtml` stuff but doesn't seem to be a pressuring issue right now.

**Comment by belucid** [comment]
+1

**Comment by martinklepsch** [comment]
`goog.dom.safeHtmlToNode` is supposed to replace `goog.dom.htmlToDocumentFragment` but doesn't take a string argument. We need a stringToSafeHtml function but I haven't found it yet. 😛 
```js
/**
 * Converts HTML markup into a node. This is a safe version of
 * {@code goog.dom.htmlToDocumentFragment} which is now deleted.
 * @param {!goog.html.SafeHtml} html The HTML markup to convert.
 * @return {!Node} The resulting node.
 */
goog.dom.safeHtmlToNode = function(html) {
  return goog.dom.safeHtmlToNode_(document, html);
};
```
https://github.com/google/closure-library/blob/4e7d45342f7c82d82b11059618f26983be466064/closure/goog/dom/dom.js#L907-L915

**Comment by jiyinyiyong** [comment]
Isn't a string a piece of HTML?

**Comment by Deraen** [comment]
Or we could just replace cljs-logo string with DOM representation in the code and we won't need to convert strings to HTML.

**Comment by martinklepsch** [comment]
Closed via #84

**Comment by paja0012** [comment]
For my project i used
`
var htmlText = '<h1>Title</h1>';
goog.dom.safeHtmlToNode(goog.html.SafeHtml.createSafeHtmlSecurityPrivateDoNotAccessOrElse(htmlText, null))
`
